content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md

```diff
@@ -151,17 +151,17 @@ Users can adjust the following thresholds to tune image garbage collection with
 
 1. `image-gc-high-threshold`，触发镜像垃圾回收的磁盘使用率百分比。默认值为 85%。
 
-<!--
-1. `image-gc-high-threshold`, the percent of disk usage which triggers image garbage collection.
-Default is 85%.
--->
+    <!--
+    1.`image-gc-high-threshold`, the percent of disk usage which triggers image garbage collection.
```

Comment:
looks like you need to add a space after `1.`.

Reply:
I know what you mean, but if adding a space after `1.`, the display will be incorrect.
Is there a better way to fix it?
![image](https://user-images.githubusercontent.com/9354727/50428689-92d53700-08f4-11e9-83b8-2c7d0c5a2f38.png)

Reply:
https://deploy-preview-11875--k8s-v1-12.netlify.com/zh/docs/concepts/cluster-administration/kubelet-garbage-collection/

Reply:
hugo is not a perfect processor yet, so we have to live with it.
Please try move all the commented contents out of the numbered list.